src/sentry/api/urls.py

```diff
@@ -1155,7 +1155,7 @@ def create_group_urls(name_prefix: str) -> list[URLPattern | URLResolver]:
         name="sentry-api-0-organization-details",
     ),
     re_path(
-        r"^(?P<organization_id_or_slug>[^\/]+)/(?:issues|groups)/",
+        r"^(?P<organization_id_or_slug>[^\/]+)/(?P<issues_or_groups>issues|groups)/",
```

Comment:
Giving this path variable a name so that we can reference it specifically instead of the default name (`{var}`).

If changing the path definition in this way has other ramifications please LMK!

Reply:
Actually turns out that adding this means that we need to add the parameter to each corresponding view or it errors out. Going to revert to the old format and just check for `{var}`